org.qgis.qgis.json

```diff
@@ -114,8 +114,8 @@
                         {
                             "type": "git",
                             "url": "https://github.com/libspatialindex/libspatialindex.git",
-                            "commit": "175b56a9541e76982d13dbbf0c9b8f0b8d752619",
-                            "tag": "2.1.0",
+                            "commit": "4a3bd449b9f2858d31a4619ba60bae6d1d30b77d",
+                            "tag": "2.0.0",
                             "x-checker-data": {
```

Comment:
Remove the checker or otherwise we'll get it upgraded. Disappointing that neither libspatialindex nor qgis addressed this in a future-proof way. :(